pkg/lib/userexport/service.go

```diff
@@ -141,61 +141,52 @@ func convertDBUserToRecord(user *user.UserForExport, record *Record) {
 }
 
 func (s *UserExportService) ExportRecords(ctx context.Context, request *Request) (outputFilename string, err error) {
-	var total_user uint64 = 0
+	// TODO: write to a tmp file
+	writer := io.MultiWriter(io.Discard, os.Stdout)
+
+	// Bound export loop maximum count
+	const maxPageToGet = 10000
```

Comment:
i thought there is a convention for giving a bound to an infinite style loop to prevent any possibility to jump into a forever loop
if no need then let me take it away👌

Reply:
removed old loop bound